docs/coding.rst

```diff
@@ -37,6 +37,16 @@ To check for stale Python dependencies (basically ``pip list -o`` but in the Doc
 For Node packages we use `NPM <https://docs.npmjs.com/cli/v8/commands/npm-install>`_, which should already be
 installed alongside `Node.js <https://nodejs.org/>`_.
 
+Front-end Dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+We currently have a few dependencies we serve on Bedrock's front-end.
```

Comment:
Suggestion (non-blocking): is it perhaps worth calling out that these are dependencies our team maintains?

Also a note that these are all published on NPM might be nice, and can be updated using `npm install`? (link to: https://docs.npmjs.com/cli/v6/commands/npm-install)?

Reply:
Good call out 🏄🏻